src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -26,42 +23,93 @@ public class StationAcceptanceTest {
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
-        // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        // given
+        String gasan = "가산디지털단지역";
 
-        // then
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        // when
+        createStationRequest(gasan);
 
         // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        assertThat(requestListAndExtractStationInfo("name")).containsAnyOf(gasan);
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @Test
+    @DisplayName("지하철역 2개를 생성하고, 목록을 조회한다.")
+    void showStations() {
+        //given
+        String gasan = "가산디지털단지역";
+        String guro = "구로디지털단지역";
+
+        createStationRequest(gasan);
+        createStationRequest(guro);
+
+        // when
+        List<String> stationNames = requestListAndExtractStationInfo("name");
+
+        // then
+        assertThat(stationNames).containsAnyOf(gasan);
+        assertThat(stationNames).containsAnyOf(guro);
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
     // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @Test
+    @DisplayName("지하철역이 생성되고, 삭제된다.")
+    void deleteStation() {
+        //given
+        String gasan = "가산디지털단지역";
+
+        createStationRequest(gasan);
 
+        // when
+        RestAssured
+                .when()
+                .delete("/stations/" + 1)
```

Comment:
매직넘버 '1' 을 사용하기 보다 지하철역 생성 응답에서 id 값을 꺼내오면 어떨까요?
```suggestion
                .delete("/stations/" + station.getId())
```

Reply:
놓쳤던 부분인데, 언급해 주셔서 감사드립니다.

기존 `createStationRequest` 메서드에 반환 타입을 `ExtractableResponse<Response>` 로 변경해서, 진행하면 될 것 같습니다.